### ISSUE TITLE
mash screen not logging anything to stdout

### ISSUE THREAD

**Issue by tiagofilipe12** [marbl/Mash#75]
I am attempting to run the same command using mash screen in a server and in my local machine. The command is as follows: `mash screen index.msh sample.fastq`. This works just fine on the server, where I generated the msh file, but in my local PC it does not log to stdout.  I checked both stderr and stdout but none had the resulting mash results. If I run `mash dist index.msh sample.fastq` everything works fine on both machines.

If instead I generate the msh file again by running `mash sketch` on the fasta and then `mash screen generated_index.msh sample.fastq` everything works as expected.

I wonder why `mash dist` accepts having files generated on the server and `mash screen` does not. 

I generated the file in:
* Ubuntu -`4.4.0-64-generic #85-Ubuntu SMP Mon Feb 20 11:50:30 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
And tried to run in:
* Manjaro Linux (Arch-based distro) - `4.13.16-2-MANJARO #1 SMP PREEMPT Fri Nov 24 12:49:10 UTC 2017 x86_64 GNU/Linux`

Also tested, using bash and zsh and the result is the same.

**Comment by ondovb** [comment]
This may be similar to #74 and #62, although it is odd that there are no errors are warnings. If the fix for that doesn't work (re-downloading binaries), would you be able to share your input files (or other files that reproduce the problem)? You can send to me at <first>.<last>@nih.gov.

**Comment by tiagofilipe12** [comment]
Hi, the key (I think) is the msh file because any fastq file will fail with that msh. You can download it [here](https://github.com/tiagofilipe12/mash_wrapper/releases/download/v1.0.3/assembly_reference.msh).

**Comment by ondovb** [comment]
It looks like the problem is indeed with the sketch file, and specifically, the same seed issue as #62. Replacing the binary on the Ubuntu side should do it (still 2.0; we replaced it early on without version change when we noticed the issue), otherwise the next thing to try would be building from source.

**Comment by tiagofilipe12** [comment]
I will test if the same binary can run the same msh file (basically put the same binary, that you have in release 2.0, on the server and on my pc and see if I can execute with the msh file generated with this version). 
It is not like it takes to much to sketch the fasta I need, but distributing a msh file is a lot lighter than a huge fasta (and of course it saves some time for my end-users). I will keep you posted.

**Comment by tiagofilipe12** [comment]
I already have results for this test. So, I run both mash sketch to generate the msh file on my pc and on a server, then I switched the files between them and run the "foreign" msh file using mash screen for the same fastq file. The results was the expected, they both could execute. So this seems related to the fact that I downloaded both binary in different days and by chance I ended up with two different binaries with the same release tag (2.0). 
I am now able to distribute the msh file, but in future releases it would be nice to have different tags (let's say 2.0.1 and so on) for cases like this where the msh file just have to be generated once and them everything else is ran with that msh file instead of starting with fasta for the reference. 

Thanks for the help.
